tuplex/test/core/DataSetCollect.cc

```diff
@@ -16,6 +16,80 @@
 
 class DataSetTest : public PyTest {};
 
+TEST_F(DataSetTest, MixedTransformWebUI) {
```

Comment:
can we maybe spin up mongodb/python etc. to test this?

Reply:
i.e., we need to find a way to test this on the CI etc., which means installing the pip package + mongodb etc. (or do a mock)

Reply:
I agree. Maybe we could discuss this in our meeting on Monday.

Reply:
I have also removed this test.